Adds OpenTargets as From/To database

hlfernandez opened the pull request:
The OpenTargets is now available and listed in the valid databases list (https://rest.uniprot.org/configure/idmapping/fields).

multimeric commented:
This probably isn't clear from the file itself, but `unipressed/id_mapping/types.py` is auto-generated via `codegen/id_mapping/generate.py`. That needs to be re-run to ensure that there aren't any other databases that have been missed. I'm happy if you do that for this PR though.

hlfernandez commented:
Sure, I'll update the file using the `generate.py` script. Sorry for the inconvenience, I didn't notice that.

hlfernandez commented:
@multimeric I have just run the `generate.py` script and updated the file. OpenTargets was the only addition but it was added in a different position so I have updated the file, amended the commit and made a `push --force`. Please, review the pull request and tell me if I should do anything else.

multimeric commented:
That's good! It's better this way because it would pass the automated CI codegen build once I implement that. I'll fix the CI issues in a separate PR before I merge this.

multimeric commented:
Okay, I've fixed the tests. Can you please pull from master and then this should be good to merge.

hlfernandez commented:
> Okay, I've fixed the tests. Can you please pull from master and then this should be good to merge.

Done!
